stories/components/FileCollection/FileCollection.stories.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from "react";
+import { Formik, Form } from "formik";
+import * as Yup from "yup";
+import styled from "styled-components";
+
+import markdown from "./FileCollection.md";
+import { ContentProvider, FileCollection } from "../../../src";
+import { Input, InputCollection } from "../../../src";
+
+export default {
+  title: "Components/FileCollection"
+};
+
+const contentNodes = {
+  "demo.placeholder": {
+    copy: {
+      en: "placeholder value",
+      fr: "placeholder value"
+    }
+  }
+};
+
+// const theme = {
+//   colors: {
+//     primary: {
+//       grey: '#d9d9e3',
+//       medium: '#c6c6d0',
+//       light: '#dfe8ff',
+//       dark: '#2000ff',
+//     },
+//   },
+// };
+
+const theme = {
+  colors: {
+    primary: {
+      light: "#f1f1f1",
+      medium: "#d9d9e3",
+      label: "#b5b5b5",
+      dark: "#74777a"
+    },
+    secondary: {
+      medium: "#2000ff"
+    },
+    highlight: {
+      medium: "#230cc2"
+    }
+  }
+};
+
+const content = {
+  contentNodes,
+  theme
+};
+
+const FileCollectionSchema = Yup.object().shape({
+  company_names: Yup.array()
+    .required("cannot be empty array")
+    .of(Yup.string().required("required"))
+});
+
+const uploadedFiles = [
+  {
+    name: "file1.txt",
+    src: "https://www.google.ca",
+    file_type: "text/plain",
+    keep: true
+  },
+  {
+    name: "file1.txt",
+    src: "https://www.google.ca",
+    file_type: "text/plain",
+    keep: true
+  }
+];
+
+export const FileCollections = () => {
+  const fileCollectionValid = values =>
+    values.files.length > 0 ||
+    !values.uploadedFiles.every(file => file.keep === false);
```

Comment:
Same as above

Reply:
changed to `values.uploadedFiles.some(file => file.keep === true)`